package.json

```diff
@@ -49,7 +49,7 @@
     "karma-webpack": "1.8.1",
     "precommit-hook": "3.0.0",
     "string-replace-loader": "1.3.0",
-    "uglifyjs-webpack-plugin": "1.1.2",
+    "uglifyjs-webpack-plugin": "1.2.2",
```

Comment:
@damencho, we use uglifyjs-webpack-plugin in both lib-jitsi-meet and jitsi-meet and, because of the building of lib-jitsi-meet as part of `nom install` in jitsi-meet, I'd recommend keeping the common dependencies of the two projects including uglifyjs-webpack-plugin at the same version. Do you have a respective PR in jitsi-meet?

Reply:
Just created one https://github.com/jitsi/jitsi-meet/pull/2586